pkgs/applications/office/libreoffice/default.nix

```diff
@@ -63,8 +63,6 @@ in (mkDrv rec {
     "-fno-visibility-inlines-hidden" # https://bugs.documentfoundation.org/show_bug.cgi?id=78174#c10
   ];
 
-  patches = [ ./xdg-open-brief.patch ];
```

Comment:
Hm, how did it build if the patch was already upstreamed?

Reply:
You mean "wasn't upstreamed yet"?

Reply:
Ah, last Hydra build is still 7.1

But then should this be applied to fresh before updating?

Reply:
I guess this could be applied just before switching fresh to 7.2.0.

Reply:
Preferably just together with the switch…

Or maybe apply this with fresh also getting the patches, and leave a comment to drop the patches for 7.2 if you want to maximise the chances of success.

On the other hand, the patch won't apply to 7.2 anyway, so it's hard to miss (`substituteInPlace` can be missed, though)